x-pack/plugins/infra/server/graphql/log_entries/resolvers.ts

```diff
@@ -115,6 +121,12 @@ export const createLogEntriesResolvers = (libs: {
         buckets,
       };
     },
+    async logItem(source, args, { req }) {
+      if (!args.id) {
+        throw new Error('id is a required field');
+      }
```

Comment:
If we declared the `id` as required, the graphql server would check that for us.